handshake_client.go

```diff
@@ -624,6 +686,14 @@ func (hs *clientHandshakeState) serverResumedSession() bool {
 func (hs *clientHandshakeState) processServerHello() (bool, error) {
 	c := hs.c
 
+	// Process the delegated_credential extension.
+	if hs.serverHello.dc != nil {
+		if hs.serverHello.vers != VersionTLS12 {
+			c.sendAlert(alertIllegalParameter)
+			return false, errors.New("tls: ServerHello with delegated_credential extension in TLS != 1.2")
+		}
```

Comment:
Is there a reason you use an underscore in delegated_credential? (ie, did you see that pattern somewhere?) It strikes me as odd

Reply:
In the context of the TLS standard, the extension is identified by the string "delegated_credential".

Reply:
Done and done. Did `s/delegated_credential/delegated credential/g` here and below.